### DISCUSSION TITLE
Cannot find module 'react-unity-webgl' or its corresponding type declarations

### DISCUSSION THREAD

**Discussion by jack-szeto** [jeffreylanters/react-unity-webgl#396] [Help Wanted!] [answered]
hi everyone here! I just upgraded the package from version `8.8.0` to `9.0.2`. But I cannot find the module anymore.

![image](https://user-images.githubusercontent.com/47553851/173533408-5f6491a5-aac6-4438-8521-4cb3886481bf.png)

Please let me know if anyone facing the same problem and any possible solutions?

In case you want to know my setup, I'm using Unity Editor 2022.2. The `package.json` looks like this:
```json
{ // package.json
  "dependencies": {
    // ...
    "next": "^12.1.6",
    "next-auth": "^4.3.4",
    "react": "18.1.0",
    "react-dom": "18.1.0",
    "react-unity-webgl": "^9.0.2",
    // ...
  }
}
```

**Comment by jeffreylanters** [comment]
Hi. I've ran some tests and cannot seem te duplicate the problem. Looking at the NPM file explorer, all of the type declarations should be there. Maybe a little bit of a weird question, but have you tried restarting VS Code? Since there are such major changes between the versions. Keep me posted!

**Reply by jack-szeto** [reply]
hey @jeffreylanters thanks for the reply.
yes i like the changes. Really appreciate your work!

I've tried to restarting VS Code and even reinstall all the `node_modules`.
It still throwing a Server Error
```
Server Error
Error: Cannot find module 'C:\distribution\exports.js'. Please verify that the package.json has a valid "main" entry
```

Maybe because I'm using Nextjs with Typescript but I'm not sure. 

I had a look at the `node_modules/react-unity-webgl` folder. I found that there's no `index.d.ts`. I'm not so familiar with this, but if this is an issue?

![image](https://user-images.githubusercontent.com/47553851/173556591-ffa07781-566e-449d-8b6a-4a7e57bb980a.png)

**Reply by jeffreylanters** [reply]
Hi! A hot-fix is on its way. Thanks for your contribution on the problem and your patience. The problem seems to only occur in Windows based machines, this is how it slipped through.

Keep an eye out for the latest version!

**Reply by jack-szeto** [reply]
hey just get back! It works now! thank you so much!

**Reply by jeffreylanters** [reply]
Awesome! Best of luck with your project, happy coding! 🙌🏼

**Comment by jeffreylanters** [comment, marked as answer]
Hi! A hot fix for this problem has published in release [v9.0.3](https://github.com/jeffreylanters/react-unity-webgl/releases/tag/v9.0.3). Thanks for your detailed report and thinking with me! Happy coding!
